### PR TITLE
fix(api): add dig to withings lambda, fix ip logic, and parallelize …

### PR DESCRIPTION
**Pull request by Goncharo**
…reqs

refs. metriport/metriport#285

### Dependencies

N/A

### Description

add dig to withings lambda, fix ip logic, and parallelize reqs

dig is based off [this OSS layer](https://github.com/mandatoryprogrammer/dig-lambda-layer)

### Release Plan

asap

### CODE REVIEW

File: api/lambdas/withings/index.js [hunk 1]
```diff
@@ -17,26 +17,34 @@ const buildResponse = (status, body) => ({
...
   return defaultResponse();
 };
 
-const lookup = async address => {
+var lookup = async address => {
   return new Promise((resolve, reject) => {
     exec(`dig +short TXT ${address}`, (error, stdout, stderr) => {
       if (error || stderr) {
-        reject("DNS lookup failed");
+        reject(
```
Comment (leite08) [inline api/lambdas/withings/index.js:41]: miss not Sentry on these lambdas yet 💔

**Review by leite08 — APPROVED** [review]
LGTM!

It would be nice to have the layer binaries out of the repo, but we can tweak this later.

File: api/lambdas/withings/index.js [hunk 2]
```diff
@@ -17,26 +17,34 @@ const buildResponse = (status, body) => ({
...
   if (withingsWhitelistIpAddresses.includes(ipAddress)) {
     return forwardCallToServer(req);
   }
-
   console.log("Request does not include a valid Withings IP address");
   return defaultResponse();
 };
 
-const lookup = async address => {
+var lookup = async address => {
```
[the remark this answers is not in the document]
Comment (Goncharo) [reply api/lambdas/withings/index.js]: fixed

File: api/lambdas/withings/index.js (hunk 1 above)
Comment (Goncharo) [reply api/lambdas/withings/index.js:41]: haha yeah, we'll get there 🌱

**Pull request merged**